README.md

```diff
@@ -282,30 +282,14 @@ By default, laravel-spanner uses [Filesystem Cache Adapter](https://symfony.com/
 There are a few cases when a 'Session not found' error can
 [happen](https://cloud.google.com/spanner/docs/sessions#handle_deleted_sessions):
 
- - Scripts that idle too long - for example, a [Laravel queue worker](https://laravel.com/docs/9.x/queues#the-queue-work-command)
-or anything that doesn't call Spanner frequently enough (more than once an hour).
- - The session is more than 28 days old.
```

Comment:
Removed since this is [no longer true](https://github.com/googleapis/google-cloud-php/pull/5853).